binaries/runtime/src/operator/mod.rs

```diff
@@ -9,17 +9,17 @@ mod shared_lib;
 
 pub struct Operator {
     operator_task: Sender<OperatorInput>,
-    config: OperatorConfig,
+    config: OperatorDefinition,
```

Comment:
```suggestion
    definition: OperatorDefinition,
```
I feel like doing a search and replace of `config` to `definition` might be nice as it is slightly confusing to read `config().config`

Reply:
Done in 2750ea59b1114bc424875b8e80738b154d02b1ff